torch/xpu/__init__.py

```diff
@@ -232,9 +232,14 @@ def get_device_capability(device: Optional[_device_t] = None) -> Dict[str, Any]:
         Dict[str, Any]: the xpu capability dictionary of the device
     """
     props = get_device_properties(device)
-    return {
+    props_dict = {
         prop: getattr(props, prop) for prop in dir(props) if not prop.startswith("__")
```

Comment:
```suggestion
        prop: getattr(props, prop) for prop in dir(props) if not prop.startswith(("__", "_pybind11_"))
```

Or `"_pybind11_conduit_"`. This is better in case there's a v2 eventually.

Reply:
Thought it seems this should filter all bound methods, it's only supposed to be looking for data members, I'd assume?

Reply:
> Thought it seems this should filter all bound methods, it's only supposed to be looking for data members, I'd assume?

It seems so, but the solution you proposed seems robust enough to not complicate the code by adding logic to detect bound methods for now. Is it ok?

Reply:
Yes, I think so.